addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -659,6 +647,40 @@ public int getWascId() {
         return 33;
     }
 
+    private AlertBuilder createUnmatchedAlert(String param, String attack, String evidence) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_LOW)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
```

Comment:
I don't think the example should set one if the code raising the alert does not set any.

Reply:
Yup that was a leftover from the first pass. Sorry.